@re-/assert/test/metaTests/templateForSnapshots.ts

```diff
@@ -0,0 +1,27 @@
+import { assert } from "../../src/assert.js"
+import { cacheAssertions, cleanupAssertions } from "../../src/type/analysis.js"
+// prettier-ignore
+const isPrecached = process.argv.includes("--reassertTestPreCached")
+isPrecached && cacheAssertions()
+
+// type
+assert({ re: "do" }).equals({ re: "do" }).type.toString.snap()
+
+//5
+assert(5).snap()
+
+// object
+assert({ re: `do` }).snap()
+
+//inlineUpdate
+// @ts-ignore (using internal updateSnapshots hook)
+assert({ re: "dew" }, { updateSnapshots: true }).snap()
+
+//update6To5
+// @ts-ignore (using internal updateSnapshots hook)
+assert(5, { updateSnapshots: true }).snap(6)
+
+//undefined
```

Comment:
When commenting something out like this, it's usually good practice to add a note to the comment that says why it's commented out (in this case a bug) and preferably a link to a GitHub issue tracking that bug. That way it's obvious the bug is my fault from my previous work and not yours as part of this PR 🤣